Tests/SentryTests/SentrySDKIntegrationTestsBase.swift

```diff
@@ -0,0 +1,92 @@
+import Foundation
+import XCTest
+
+class SentrySDKIntegrationTestsBase: XCTestCase {
```

Comment:
`l`: Nice to create a base class, but we only use it once. What's the purpose?

Reply:
Every integration code that we need to check the captured event we have to duplicate code to do it

Reply:
Makes sense thanks.